sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/AbstractSparkSQLDriver.scala

```diff
@@ -79,3 +91,6 @@ private[hive] abstract class AbstractSparkSQLDriver(
     tableSchema = null
   }
 }
+
+private[hive] case class CommandProcessorResponseWrapper (rc : CommandProcessorResponse,
```

Comment:
This is not the right style: https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide

```scala
private[hive] case class CommandProcessorResponseWrapper(
    rc: CommandProcessorResponse,
    cause: Throwable)
```